README.md

```diff
@@ -1,18 +1,68 @@
 # primeui-tester
-primeui-tester is a API which helps you test your JSF-Page containing primefaces widgets. The API provides you methods to interact with your widgets and check if it's in the condition you expected it to be. 
+primeui-tester is a API which helps you test your JSF-Page containing primefaces
+widgets. The API provides you methods to interact with your widgets and check if
+it's in the condition you expected it to be. 
 
 # How to use in your project
-Simply add this library to your dependencies in the pom.xml
+The primeui-tester runs with [Selenide](https://selenide.org/),
+[Selenium](https://selenium.dev/projects/) and [JUnit
+5](https://junit.org/junit5/). Simply add this library to your dependencies in
+the pom.xml:
 
-    <dependencies>
-    ...
-      <dependency>
-        <groupId>com.axonivy.ivy.supplements</groupId>
-    	  <artifactId>primeui-tester</artifactId>
-    	  <version>0.0.5-SNAPSHOT</version>
-        <scope>tests</scope>
-      </dependency>
-    </dependencies>
+```xml
+<dependencies>
+...
+  <dependency>
+    <groupId>com.codeborne</groupId>
```

Comment:
can make the required deps smaller?
improve dependency pain by declaring optional/test scoped deps in primeui-tester?
shouldn't some deps of prime-ui be optional as well?